src/organisation/store/actions/organisation.actions.ts

```diff
@@ -13,7 +13,7 @@ export class LoadOrganisation {
 
 export class LoadOrganisationSuccess implements Action {
   readonly type = LOAD_ORGANISATION_SUCCESS;
-  constructor(public payload: Organisation) { }
+  constructor(public payload: any) { }
```

Comment:
why did you set to any?

Reply:
good spot - we're unsure on the model for organisation  - Example the format of the address property, is it a single string?  or multiple property as required for street, town,postcode - hence the api needs testing by posting a addresss - however there is a bug thats preventing this and this has been assigned to Tabby for now - will revisit and update <any>  in future